ceno_zkvm/src/bin/e2e.rs

```diff
@@ -41,6 +43,11 @@ struct Args {
     /// The preset configuration to use.
     #[arg(short, long, value_enum, default_value_t = Preset::Ceno)]
     platform: Preset,
+
+    /// The private input or hints. This is a raw file mounted as a memory segment.
```

Comment:
I'd say the raw file is 'mapped' as a memory segment.  Mounting is generally used for file systems.

Reply:
Why do you give a thumbs up, but then don't do anything about it?

Reply:
I meant to include it but I was focused on making the feature work.

Reply:
Ok, no worries, I'll create a PR.

Reply:
https://github.com/scroll-tech/ceno/pull/633